Common/Global.cs

```diff
@@ -429,6 +445,22 @@ public enum OptionStyle
     }
 
     /// <summary>
+    /// Specifies the type of settlement in derivative deals 
+    /// </summary>
+    public enum SettlementType
+    {
+        /// <summary>
+        /// Physical delivery of the underlying security 
+        /// </summary>
+        PhysicalDelivery, 
```

Comment:
heheh IB doesn't even accept orders on GC front month contract.. due to this exact idea :) They don't want you to take delivery!